docs/book/cookbook/aot-guide.md

```diff
@@ -127,6 +127,7 @@ class ConfigProvider
                 'aot' => [
                     'namespace' => __NAMESPACE__ . '\\Generated',
                     'directory' => __DIR__ . '/../gen',
+                    'logger' => \Psr\Log\LoggerInterface::class,
```

Comment:
This cookbook recipe does not use the logger therefore is the configuration unnecessary and misleading.